provider/datagen/tests/make-testdata.rs

```diff
@@ -2,21 +2,30 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
-use icu_datagen::fs_exporter::serializers::{Json, Postcard};
+use icu_datagen::fs_exporter::serializers::Json;
 use icu_datagen::fs_exporter::*;
 use icu_datagen::prelude::*;
 use icu_provider::datagen::*;
 use icu_provider::prelude::*;
-use std::collections::BTreeSet;
+use std::collections::{BTreeMap, BTreeSet};
+use std::fs::File;
+use std::io::Write;
 use std::path::Path;
+use std::sync::Mutex;
 
+// Heap measurements don't work in a parallel environment.
+#[cfg(not(feature = "rayon"))]
 #[global_allocator]
 static ALLOC: dhat::Alloc = dhat::Alloc;
 
 include!("data/locales.rs.data");
 
 #[test]
-fn generate_fs_and_verify_zero_copy() {
+fn generate_json_and_verify_postcard() {
+    // don't drop to avoid dhat from printing stats at the end
```

Comment:
nit: the test should probably warn in rayon mode.

Reply:
how can a test warn? there's no output unless it fails

Reply:
ah fair